frontend/src/components/messages/NewRequestDialog.vue

```diff
@@ -185,17 +389,38 @@ export default {
         this.setUpDefaultNewRequestModel()
       },
     },
+    'newRequestModel.subCategories': {
+      handler() {
+        if (this.changeTypeClass === 'change-type-required' && this.isAnySubCategoryChecked) {
+          this.changeTypeClass = ''
+        }
+      },
+      deep: true
+    }
   },
   created() {
+    this.PHONE_FORMAT = PHONE_FORMAT
+    this.EMAIL_FORMAT = EMAIL_FORMAT
+    this.AUTO_REPLY_MESSAGE = 'Your change request is complete.'
+    this.requestSubCategorySortOrder = [
+      "Organization Details",
+      "Facility Details",
+      "Organization phone/email",
+      "Facility phone/email",
+      "Add/change a licence",
+      "Other"
+    ]
+    this.sortedRequestSubCategories = this.sortRequestSubCategories(this.requestSubCategories, this.requestSubCategorySortOrder)
```

Comment:
Since we hardcoded the sub-category name, we will need a code change if the Business change the name of those sub-categories (e.g.: Other -> Others). If we really need to maintain the order of subcategories, we should check with CRM team to see if they have any property that we can use to sort (e.g.: similar to what they did for categories), then we can include orderby id in our backend query

![image](https://github.com/bcgov/ecc-ofm/assets/119005543/65052822-4bc2-41ba-9ee9-219244740778)

Reply:
That makes sense, can we move this PR forward for QA and i'll work on circling this back in.. since its dependent on CRM it may takes some time but i'm sure we can get it in.

Reply:
sounds good to me!